src/capi/azext_capi/custom.py

```diff
@@ -520,11 +531,114 @@ def create_workload_cluster(  # pylint: disable=unused-argument,too-many-argumen
 
     # Wait for all nodes to be ready before returning
     with Spinner(cmd, "Waiting for workload cluster nodes to be ready", "✓ Workload cluster is ready"):
-        wait_for_ready(workload_cfg)
+        wait_for_nodes(workload_cfg)
 
+    if pivot:
+        pivot_cluster(cmd, workload_cfg)
     return show_workload_cluster(cmd, capi_name)
 
 
+def pivot_cluster(cmd, target_cluster_kubeconfig):
+
+    logger.warning("Starting Pivot Process")
+    begin_msg = "Installing CAPZ components in target management cluster"
```

Comment:
```suggestion
    begin_msg = "Installing Cluster API components in target management cluster"
```

is slightly more accurate. although I realize that function name is misleading...

Reply:
I am not sure I am following. Do you mind elaborating about the function name?

Reply:
I just meant that the function we're calling is called `_install_capz_components` which makes it seem like we're installing just capz, even though what we're doing is initializing the management cluster with all Cluster API providers, including CAPZ